haskell/src/Monocle/Prelude.hs

```diff
@@ -553,3 +559,26 @@ data AnyJSON = AnyJSON
 
 instance FromJSON AnyJSON where
   parseJSON _ = pure AnyJSON
+
+genRandomBS :: IO ByteString
```

Comment:
That function looks suspicious, but I guess it works and we could rewrite it later.

Reply:
I wrote something much simpler.